### PR TITLE
Initial version of the give_total_forms shortcode

### PR DESCRIPTION
**Pull request by matheuswd**
This is an initial version of the custom shortcode that checks how many donation forms are published and allows the admin to select to which page the donor will be redirected to select the donation form they want to donate to.

### CODE REVIEW

File: custom-shortcodes/give-total-forms.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	$atts = shortcode_atts( array(
+		'donation_forms_page' => false,
+	), $atts, 'give_total_forms' );
+
+	$donation_forms_page = '';
+
+	if ( $atts['donation_forms_page'] != false ) {
+		$donation_forms_page = '<a href="' . $atts['donation_forms_page'] . '"> Check out our campaigns! <a>';	
+	}
```
Comment (jonwaldstein) [inline custom-shortcodes/give-total-forms.php:23]:
```suggestion
	if ( $atts['donation_forms_page_id']) {
        	$link = get_permalink($atts['donation_forms_page_id']);
        	
		$donation_forms_page = '<a href="' . $link . '"> Check out our campaigns! <a>';	
	}
```

File: custom-shortcodes/give-total-forms.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ *  [give_total_forms] - displays a paragraph containing the number of donation forms you have on your website. The message will say: "We have 64 campaigns that you can donate to!"
+ * 
+ *  [give_total_forms donation_forms_page="https://example.com/donations/"] - displays the paragraph containing the number of donation forms you have on your website and a link pointing to https://example.com/donations/" where your donors can see your donation forms and donate.
+ */
+
+function my_give_display_number_of_forms( $atts ) {
+	$number_of_donation_forms = wp_count_posts( 'give_forms' )->publish;
+
+	$atts = shortcode_atts( array(
+		'donation_forms_page' => false,
```
Comment (jonwaldstein) [inline custom-shortcodes/give-total-forms.php:16]:
```suggestion
		'donation_forms_page_id' => false,
```

File: custom-shortcodes/give-total-forms.php (hunk 1 above)
Comment (jonwaldstein) [reply custom-shortcodes/give-total-forms.php:23]: @matheuswd I would suggest getting the permalink from the id so the link is dynamic.

File: custom-shortcodes/give-total-forms.php (hunk 2 above)
Comment (jonwaldstein) [reply custom-shortcodes/give-total-forms.php:16]: @matheuswd I would suggest changing this to an ID so it's more clear & we can make the link dynamic

**Review by jonwaldstein — CHANGES_REQUESTED** [review]
@matheuswd nice job! just wanted to make a suggestion to change the `donation_forms_page` argument to an ID.

**Review by matheuswd — COMMENTED** [review]
👍

**Review by jonwaldstein — APPROVED** [review]
Great job!

**Pull request merged**